src/gerbil/test/c3-test.ss

```diff
@@ -1,35 +1,94 @@
 ;; -*- Gerbil -*-
 ;;; © fare@tunes.org
 ;;;; Testing the c3 linearization algorithm
+prelude: "../prelude/core"
 
 (export c3-test)
 
 (import
   ;;  :gerbil/runtime/c3
   (only-in :gerbil/runtime/mop type-descriptor-all-slots)
+  (only-in :std/misc/hash hash-ensure-ref)
   (only-in :std/sugar defrule)
-  (only-in :std/test test-suite test-case check check-exception))
+  (only-in :std/test test-suite test-case check check-exception)
+  (only-in :std/values first-value))
 
 (define gerbil/runtime 'gerbil/runtime)
 (define :gerbil/core ':gerbil/core)
+;;(include "../../gerbil/runtime/util.ss")
 (include "../../gerbil/runtime/c3.ss")
```

Comment:
we should be able to just import this for running the tests after everything is done.